### PR TITLE
fix(docs): wrong arguments in models' getDevice request

### PR DESCRIPTION
**Pull request by Xamez**
## What does this PR do ?

Fixes incorrect arguments in the request to retrieve a device's model.

## ToDos
- [x] getAsset: wrong arguments & wrong controller & wrong slug route argument
- [x] getMeasure: wrong argument (type) & wrong slug route argument

### CODE REVIEW

**Comment by GuinierRemi** [comment]
Fixed both Todos, @tdislay can you please check ?

File: doc/2/controllers/models/get-device/index.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Gets an device model.
 ### HTTP
 
 ```http
-URL: http://kuzzle:7512/_/device-manager/models/device/:_id
+URL: http://kuzzle:7512/_/device-manager/models/device/:model
```
Comment (tdislay) [inline doc/2/controllers/models/get-device/index.md:19]: Actually, the slug `_id` was okay. [See this](https://github.com/kuzzleio/kuzzle-device-manager/blob/master/lib/modules/model/ModelsController.ts#L49C52-L49C62)

File: doc/2/controllers/models/get-device/index.md [hunk 2]
```diff
@@ -26,17 +26,15 @@ Method: GET
...
 }
 ```
 
 ---
 
 ## Arguments
 
-- `engineId`: engine id
-- `_id`: device model id
+- `model`: device model id
```
Comment (tdislay) [inline doc/2/controllers/models/get-device/index.md:37]: It's not really an id
```suggestion
- `model`: device model
```

File: doc/2/controllers/models/get-measure/index.md [hunk 3]
```diff
@@ -26,16 +26,15 @@ Method: GET
...
 }
 ```
 
 ---
 
 ## Arguments
 
-- `engineId`: engine id
-- `_id`: measure model id
+- `type`: The type of measure you want to retreive.
```
Comment (tdislay) [inline doc/2/controllers/models/get-measure/index.md:37]: Suggestion in order to stay consistent.
```suggestion
- `type`: measure type
```

File: doc/2/controllers/models/get-device/index.md [hunk 4]
```diff
@@ -26,17 +26,15 @@ Method: GET
 {
   "controller": "device-manager/models",
   "action": "getDevice",
-  "engineId": "<engineId>",
-  "_id": "<modelId>"
+  "model": "<modelId>"
```
Comment (tdislay) [inline doc/2/controllers/models/get-device/index.md:29]:
```suggestion
  "model": "<device model>"
```

**Review by tdislay — APPROVED** [review]

**Pull request merged**